data/distro.yml

```diff
@@ -564,30 +564,41 @@
       <p>Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>
     </ol>
 
-- name: "SulinOS"
-  logo: "sulinos.svg"
```

Comment:
Please also remove the actual sulinos.svg file.

Reply:
done